### PR TITLE
Add docs for APM agent setup with ECS env.

### PR DESCRIPTION
**Pull request by umaannamalai**
This PR adds a guide for setting up language agents/ the infra agent within an ECS/ Fargate environment.

### CODE REVIEW

**Review by jaffinito — APPROVED** [review]
This looks great!  There a few spots where you could replace the full docs URL withe shorthand version, but I don't think that should hold up approval.

File: src/content/docs/infrastructure/elastic-container-service-integration/introduction-amazon-ecs-integration.mdx [hunk 1]
```diff
@@ -62,3 +62,8 @@ To install, see our [install integration docs](/docs/install-ecs-integration).
 ## Check the source code [#source-code]
 
 This integration is open source software. This means you can [browse its source code](https://github.com/newrelic/nri-ecs) and send improvements, or create your own fork and build it.
+
+
+## Monitoring with APM language agents
```
Comment (akristen) [inline src/content/docs/infrastructure/elastic-container-service-integration/introduction-amazon-ecs-integration.mdx:67]:
```suggestion
## Monitor with APM language agents [#monitor-with-apm]
```

File: src/content/docs/infrastructure/elastic-container-service-integration/introduction-amazon-ecs-integration.mdx [hunk 2]
```diff
@@ -62,3 +62,8 @@ To install, see our [install integration docs](/docs/install-ecs-integration).
...
+
+You can also set up the APM language agents to run in your ECS/ Fargate environment. For more details, please see the [APM Agent in ECS Setup Guide](/docs/infrastructure/elastic-container-service-integration/monitor-ecs-with-apm-agents).
```
Comment (akristen) [inline src/content/docs/infrastructure/elastic-container-service-integration/introduction-amazon-ecs-integration.mdx:69]:
```suggestion
You can also set up the APM language agents to run in your ECS Fargate environment. For more details, please see the [APM Agent in ECS Setup Guide](/docs/infrastructure/elastic-container-service-integration/monitor-ecs-with-apm-agents).
```

File: src/content/docs/infrastructure/elastic-container-service-integration/monitor-ecs-with-apm-agents.mdx [hunk 3]
```diff
@@ -0,0 +1,280 @@
+---
+title: Monitor Amazon ECS environments with APM language agents
```
Comment (akristen) [inline src/content/docs/infrastructure/elastic-container-service-integration/monitor-ecs-with-apm-agents.mdx:2]: We need to add this to our yml file `apm.yml`

File: src/content/docs/infrastructure/elastic-container-service-integration/monitor-ecs-with-apm-agents.mdx [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+/>
+
+<figcaption>
+  This is an example of how the service map would be populated if you run a language agent and the infrastructure agent together in a Fargate environment.
+</figcaption>
+
+For more information, please see our [ECS integration install guide](https://docs.newrelic.com/docs/infrastructure/elastic-container-service-integration/install-ecs-integration/).
+
+
+Example full task definition file:
```
Comment (akristen) [inline src/content/docs/infrastructure/elastic-container-service-integration/monitor-ecs-with-apm-agents.mdx:160]: Can we add a code comment somewhere that identifies the changes specific to setting up a sidecar container?

Comment (umaannamalai) [reply src/content/docs/infrastructure/elastic-container-service-integration/monitor-ecs-with-apm-agents.mdx]: This task definition file was meant to show the full setup of the sidecar and the agent. I can add it under a new main heading to clarify that

**Review by akristen — APPROVED** [review]

**Pull request merged**